bitnami/kong/templates/kong-prometheus-rbac.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if and .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ template "kong.fullname" . }}-prometheus
+  namespace: {{ .Release.Namespace }}
+rules:
+- apiGroups: [""]
+  resources: ["endpoints", "services", "pods"]
+  verbs: ["get", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "kong.fullname" . }}-prometheus
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ template "kong.fullname" . }}-prometheus
+subjects:
+  {{- if .Values.metrics.serviceMonitor.namespace }}
+  - namespace: {{ .Values.metrics.serviceMonitor.namespace }}
+  {{- else }}
+  - namespace: {{ .Release.Namespace }}
+  {{- end }}        
+  kind: ServiceAccount
+  {{- if .Values.metrics.serviceMonitor.serviceAccount }}
+  name: {{ .Values.metrics.serviceMonitor.serviceAccount }}
+  {{- else }}
+  name: {{ template "kong.serviceAccount" . }}
+  {{- end }}
+{{- end }}
```

Comment:
Please split this YAML into two files: `role.yaml` and `rolebinding.yaml`
Just for consistency with other charts.

Reply:
done